src/js/main.js

```diff
@@ -363,6 +363,23 @@ if (document.body.dataset.page === "integrations") {
   }
 }
 
+// Add underline class to menu links on Mobile
+
+function setActiveMenuItem() {
+  const currentPath = window.location.pathname;
+  const menuItems = document.querySelectorAll("nav a");
```

Comment:
Btw this matches much more elements, let's add an id to the `<nav>` that we are interested in to make it more specific?

Reply:
Done! =D